main.js

```diff
@@ -5,17 +5,24 @@ let count = 0;
 
 
 const expandMethods = {
-	toggleContent: function (state) {
+	toggleAccordionContent: function (state) {
```

Comment:
It would be good to put in tests for the new methods before merging this instead of leaving it until we have to write an entire set of tests for the component.